roles/pre_tasks/defaults/main.yml

```diff
@@ -5,6 +5,9 @@
 #               Tower Installation Vars                    #
 ############################################################
 
+# If you are running AAP2.X+ use automationcontroller any earlier version use tower
+tower_or_aap: automationcontroller
```

Comment:
Sorry to be a pain, I think I would prefer it as a true/false value.

Perhaps `legacy_tower` and set the default to false. Then in the template you can just use some logic to set either [tower] or [automationcontroller].

The reason behind that is that it will just be easier for the end user (especially given there are only two possible values)

Reply:
no problem